settings.gradle.kts

```diff
@@ -24,7 +24,7 @@ pluginManagement {
 }
 
 plugins {
-    id("com.gradle.enterprise").version("3.3.4")
+    id("com.gradle.enterprise").version("3.4")
     id("com.gradle.enterprise.gradle-enterprise-conventions-plugin").version("0.3")
```

Comment:
@jonnybbb please also update the conventions plugin to `0.6`. I forgot to tell you that part. Sorry for that.

Reply:
Just added it via a suggestion.